pages/[user].js

```diff
@@ -59,10 +59,21 @@ const UserPage = () => {
           <a>{user.statistics.subscribers} subscribers</a>{' '}
 
           {user.type === 'user' && <>
-            <a>{user.statistics.subscriptions} subscriptions</a>{' '}
-            <a>{user.statistics.posts} posts</a>{' '}
-            <a>{user.statistics.comments} comments</a>{' '}
-            <a>{user.statistics.likes} likes</a>{' '}
+            <Link href={''}>
+              <a>{user.statistics.subscriptions} subscriptions</a>
+            </Link>
+            {' '}
+            <Link href={'/' + username}>
+              <a>{user.statistics.posts} posts</a>
+            </Link>
+            {' '}
+            <Link href={'/' + username + '/comments'}>
+              <a>{user.statistics.comments} comments</a>
+            </Link>
+            {' '}
+            <Link href={''}>
+              <a>{user.statistics.likes} likes</a>
+            </Link>
```

Comment:
Why this one is empty? Let's include /likes in this PR, it's going to be pretty much the same.